src/matchers/anyString.js

```diff
@@ -0,0 +1,20 @@
+import Matcher from './matcher';
+class AnyString extends Matcher {
+  constructor(stringToMatch){
```

Comment:
The logic here should be reversed. the consumption pattern is 
```js
ninja.getArsenal.when(matchers.anyString()).then.return('Die in a fire');
ninja.getArsenal('a');
//when ninja.getArsenal is invoked the matcher.matches function is invoked with the runtime value... in this case 'a'
//new AnyString().matches('a')
//is invoked to figure out if we should to the then clause
```
The way its currently written you'd have this behavior
```js
ninja.getArsenal.when(matchers.anyString('a')).then('Die in a fire');
ninja.getArsenal();
//Will return 'Die in a fire', because anyString('a').matches() will always return true.
```

So your matches function is exactly right, but it should take in the string there instead of the constructor.